build.gradle

```
@@ -4,14 +4,12 @@ plugins {
     id'application'
     id'com.github.johnrengelman.shadow' version '5.2.0'
     //noinspection GroovyAssignabilityCheck
-    id 'org.springframework.boot' version '2.5.5'
-    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
     id 'java'
 }
 
 mainClassName = 'com.severalcircles.flames.system.Flames'
 
-version '4.2.1'
+version 'SNAPSHOT-4.3a'
```

SNAPSHOT should be removed from the version number before release
```suggestion
version '4.3'
```